sonar-core/src/main/java/org/sonar/core/user/UserDto.java

```diff
@@ -88,11 +97,39 @@ public String getScmAccounts() {
     return scmAccounts;
   }
 
-  public UserDto setScmAccounts(@Nullable String scmAccounts) {
-    this.scmAccounts = scmAccounts;
+  public List<String> getScmAccountsAsList() {
+    return decodeScmAccounts(scmAccounts);
+  }
+
+  /**
+   * List of SCM accounts separated by '|'
+   */
+  public UserDto setScmAccounts(@Nullable String s) {
```

Comment:
Indeed, in such case maybe we should call this methods getRawXXX and setRawXXX ? It would make it more easy to understand.